src/renderer/components/SettingsPage.tsx

```diff
@@ -852,16 +1087,35 @@ export default class SettingsPage extends React.PureComponent<Record<string, nev
                     value={this.state.logLevel}
                     onChange={this.handleChangeLogLevel}
                 >
-                    <option value='error'>{'Errors (error)'}</option>
-                    <option value='warn'>{'Errors and Warnings (warn)'}</option>
-                    <option value='info'>{'Info (info)'}</option>
-                    <option value='verbose'>{'Verbose (verbose)'}</option>
-                    <option value='debug'>{'Debug (debug)'}</option>
-                    <option value='silly'>{'Finest (silly)'}</option>
+                    <option value='error'>
+                        {intl.formatMessage({id: 'renderer.components.settingsPage.loggingLevel.level.error', defaultMessage: 'Errors (error)'})}
```

Comment:
Does FormattedMessage not work here? As far as I know, it just renders plain text without any tags around it

Reply:
It actually renders as a `span` by default, which I think is defined [here](https://github.com/mattermost/desktop/pull/2190/files#diff-b44a65967cbe699fe5946cb191a63b9d228d678e3a42fa4a305b5975d2363c42R48)

We do the same in the webapp, so I copied the behaviour here.

Reply:
Oh interesting. I thought it didn't render a tag because it doesn't let you apply styles or a className to FormattedMessage directly.

Reply:
I think you can override it if you choose, but by default you'll get a `span`. Which was very convenient for me since most of my text was wrapped in `span` anyway.